yqlplus_engine/src/main/java/com/yahoo/yqlplus/engine/internal/bytecode/types/gambit/ExpressionHandler.java

```diff
@@ -879,6 +879,27 @@ public void generate(CodeEmitter code) {
             };
         }
     }
+    
+    private static boolean isJavaFieldName(String fieldName) {
+      if (fieldName.isEmpty()) {
+        return false;
+      } else {
+        char[] characters = fieldName.toCharArray();
```

Comment:
fieldName.charAt will index a string -- so using fieldName.charAt(i) in place of characters[i] would avoid needing the array.